frontend/src/main/java/com/java/academy/gym/util/JsonUtil.java

```diff
@@ -0,0 +1,17 @@
+package com.java.academy.gym.util;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public final class JsonUtil {
+
+    private JsonUtil() {
+    }
+
+    public static byte[] toJson(Object object) throws JsonProcessingException {
```

Comment:
Maybe gson library could be better?

Reply:
fasterxml comes with spring so I think you can stay with it